### ISSUE TITLE
`make populate_data` fails

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#9506]
(Skipping irrelevant output)

```
python manage.py generate_default_addons_for_frontend
{"EnvVersion": "2.0", "Type": "z.startup", "Severity": 6, "Fields": {"msg": "Set RECURSION_LIMIT to 10000", "remoteAddressChain": "", "uid": ""}, "Timestamp": 1537802148116235008, "Logger": "http_app_addons", "Hostname": "1addf5ce647b", "Pid": 371}
{"EnvVersion": "2.0", "Type": "raven.contrib.django.client.DjangoClient", "Severity": 7, "Fields": {"msg": "Configuring Raven for host: None", "remoteAddressChain": "", "uid": ""}, "Timestamp": 1537802148130449920, "Logger": "http_app_addons", "Hostname": "1addf5ce647b", "Pid": 371}
{"EnvVersion": "2.0", "Type": "raven.contrib.django.client.DjangoClient", "Severity": 6, "Fields": {"msg": "Raven is not configured (logging is disabled). Please see the documentation for more information.", "remoteAddressChain": "", "uid": ""}, "Timestamp": 1537802148131510016, "Logger": "http_app_addons", "Hostname": "1addf5ce647b", "Pid": 371}
{"EnvVersion": "2.0", "Type": "raven.base.Client", "Severity": 7, "Fields": {"msg": "Configuring Raven for host: None", "remoteAddressChain": "", "uid": ""}, "Timestamp": 1537802148261197056, "Logger": "http_app_addons", "Hostname": "1addf5ce647b", "Pid": 371}
{"EnvVersion": "2.0", "Type": "raven.base.Client", "Severity": 6, "Fields": {"msg": "Raven is not configured (logging is disabled). Please see the documentation for more information.", "remoteAddressChain": "", "uid": ""}, "Timestamp": 1537802148262240000, "Logger": "http_app_addons", "Hostname": "1addf5ce647b", "Pid": 371}
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 206, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 40, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/code/src/olympia/landfill/management/commands/generate_default_addons_for_frontend.py", line 8, in <module>
    from olympia.landfill.serializers import GenerateAddonsSerializer
  File "/code/src/olympia/landfill/serializers.py", line 12, in <module>
    from fxa.constants import ENVIRONMENT_URLS
ImportError: No module named fxa.constants
Makefile-docker:88: recipe for target 'populate_data' failed
make: *** [populate_data] Error 1
```

**Comment by eviljeff** [comment]
This is because the `fxa` package is only listed in the `ui-tests.txt` file.  If we can't work around needing it (and I don't think we can) then we should move that dep to `requirements/tests.txt`

**Comment by tb120** [comment]
If it helps, I added the below to `requirements/tests.txt` and successfully completed the `docker_initialize` task:
```
 PyFxA==0.6.0 \
      --hash=sha256:e37149cf27e26eed2b321389d313c876ffbbdd7868ec7f0694a8989847b9c8da \
      --hash=sha256:d511b6f43a9445587c609a138636d378de76661561116e1f4259fcec9d09b42b
 pytest-fxa==1.4.0 \
      --hash=sha256:778dfdb019f1e0af8744704fe5f7ac5c08fd5d45ff054023b0a18d5f99d737f1 \
      --hash=sha256:b75967e74e9b2f3ffa5558421fdf61c7fff5948fc9d7e357e7147c682988ecc1
 PyBrowserID==0.14.0 \
      --hash=sha256:8e237d6a2bc9ead849a4472a84d3e6a9309bec99cf8e10d36213710dda8df8ca \
      --hash=sha256:6c227669e87cc25796ae76f6a0ef65025528c8ad82d352679fa9a3e5663a71e3
 hawkauthlib==2.0.0 \
      --hash=sha256:935878d3a75832aa76f78ddee13491f1466cbd69a8e7e4248902763cf9953ba9 \
      --hash=sha256:effd64a2572e3c0d9090b55ad2180b36ad50e7760bea225cb6ce2248f421510d
 WebOb==1.8.2 \
     --hash=sha256:263690003a3e092ca1ec4df787f93feb0004e39d7bac9cba2c19a552c765894b \
     --hash=sha256:1fe722f2ab857685fc96edec567dc40b1875b21219b3b348e58cd8c4d5ea7df3

```

**Comment by eviljeff** [comment]
@tb120 did you need _all_ of those dependencies?  I thought one would have been enough.

**Comment by tb120** [comment]
@eviljeff I added the three fxa modules and the make task still failed. I added `hawkauthlib` and it failed as well until I finally added `Web0b`. I was just trying to get it to work so I didn't investigate the source fully but I do know it didn't work until the latter 2 modules were added. 

I was a bit surprised because one of the latter two modules are a dependency themselves, which I thought would have been resolved when the initial module was added. I think that i saw you have a more formal method for adding dependencies but I was simply copy and pasting into the file.
